docs/overrides/partials/header.html

```diff
@@ -0,0 +1,141 @@
+<!--
+  Copyright (c) 2016-2024 Martin Donath <martin.donath@squidfunk.com>
```

Comment:
I assume this came from the mkdocs-material repo somewhere? Or is it a copy of something we already have included? Either way, please include a reference to the source. If it's a modified version of something from upstream, please indicate where the modification is.

Reply:
Thanks, the file already contained a link to the upstream, but not at the top of the file.
It should be clearer now.